src/libmongoc/tests/unified/operation.c

```diff
@@ -760,7 +760,9 @@ operation_distinct (test_t *test,
                         "key",
                         BCON_UTF8 (field_name),
                         "query",
-                        &filter);
+                        "{",
+                        &filter,
+                        "}");
```

Comment:
From the MongoDB's docs [here](https://docs.mongodb.com/manual/reference/command/distinct/index.html) the `query` should be a document. Previously, we were setting it as a string. 

See the docs for [BCON_NEW](http://mongoc.org/libbson/current/creating.html#simplified-bson-c-object-notation) for an example on how to use this function.